region/crateria/east/Bowling Alley Path.json

```diff
@@ -149,6 +149,10 @@
           "position": "bottom"
         }
       },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
```

Comment:
Good catch, just tested and it looks possible (hard to be 100% sure without actually making it a green door). It seems with controller you have to use a piano grip and carefully script the sequence of inputs. I'm not practiced at this kind of thing but it seems extremely tricky, and if possible it would be good to put a higher tech requirement on it (to put it in Extreme); was only able to get it once after a hundred tries or so. I added a FIXME.